### PR TITLE
adding ethersjs and updating accounts in genesis file

### PR DESCRIPTION
**Pull request by joshuafernandes**

### CODE REVIEW

File: files/besu/smart_contracts/scripts/keys.js [hunk 1]
```diff
@@ -33,5 +33,16 @@ module.exports = {
...
-  }
+  },
+  accounts: {
+    "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73" : {
+      "privateKey" : "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63",
+    },
+    "0x627306090abaB3A6e1400e9345bC60c78a8BEf57" : {
+      "privateKey" : "c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3",
+      },
+    "f17f52151EbEF6C7334FAD080c5704D77216b732" : {
```
Comment (Ezzahhh) [inline files/besu/smart_contracts/scripts/keys.js:44]: Maybe add "0x" in front of this one as well?

File: files/goquorum/smart_contracts/scripts/keys.js [hunk 2]
```diff
@@ -39,5 +39,16 @@ module.exports = {
...
-  }
+  },
+  accounts: {
+    "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73" : {
+      "privateKey" : "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63",
+    },
+    "0x627306090abaB3A6e1400e9345bC60c78a8BEf57" : {
+      "privateKey" : "c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3",
+      },
+    "f17f52151EbEF6C7334FAD080c5704D77216b732" : {
```
Comment (Ezzahhh) [inline files/goquorum/smart_contracts/scripts/keys.js:50]: Same as other comment

**Review by Ezzahhh — APPROVED** [review]
LGTM

File: files/besu/smart_contracts/scripts/keys.js (hunk 1 above)
Comment (joshuafernandes) [reply files/besu/smart_contracts/scripts/keys.js:44]: Nice catch :)

File: files/goquorum/smart_contracts/scripts/keys.js (hunk 2 above)
Comment (joshuafernandes) [reply files/goquorum/smart_contracts/scripts/keys.js:50]: :+1:

**Pull request merged**